### PR TITLE
Removed Direct Debit sections and mentions.

### PR DESCRIPTION
**Pull request by arebra**
Direct Debit sections and mentions are gone.

### CODE REVIEW

File: _includes/settlement-reconciliation.md [hunk 1]
```diff
@@ -49,15 +49,15 @@ balance report summary is equivalent to the disbursement on the bank statement
 ##### Sales specification
 
 Provides a specification over sales for the given period. The sales total is
-specified per payment area (`CreditCard, Direct Debit, Factoring`) and
+specified per payment area (`CreditCard, Factoring`) and
```
Comment (asbjornu) [inline _includes/settlement-reconciliation.md]: Which payment area is `Factoring`? Is that relevant for users of eCom products?

```suggestion
specified per payment area (`CreditCard`, `Factoring`) and
```

File: _includes/settlement-reconciliation.md [hunk 2]
```diff
@@ -49,15 +49,15 @@ balance report summary is equivalent to the disbursement on the bank statement
...
 underlying payment instruments. Each sales row specify Quantity, Sum sales and
```
Comment (asbjornu) [inline _includes/settlement-reconciliation.md]:
```suggestion
underlying payment instruments. Each sales row specify `Quantity`, `Sum sales` and
```

File: _includes/settlement-reconciliation.md [hunk 3]
```diff
@@ -49,15 +49,15 @@ balance report summary is equivalent to the disbursement on the bank statement
...
 Amount to pay out, the last one is only eligble **if Swedbank Pay handles the
```
Comment (asbjornu) [inline _includes/settlement-reconciliation.md]:
```suggestion
`Amount to pay out`, the last one is only eligble **if Swedbank Pay handles the
```

File: _includes/settlement-reconciliation.md [hunk 4]
```diff
@@ -49,15 +49,15 @@ balance report summary is equivalent to the disbursement on the bank statement
...
 Settlement process**.
 
 ##### Fees specification
 
 Provides a specification over fees for the given period. The fees total is
-specified per payment area (`CreditCard, Direct Debit, Factoring`) and
+specified per payment area (`CreditCard, Factoring`) and
```
Comment (asbjornu) [inline _includes/settlement-reconciliation.md]:
```suggestion
specified per payment area (`CreditCard`, `Factoring`) and
```

File: _includes/settlement-reconciliation.md [hunk 5]
```diff
@@ -49,15 +49,15 @@ balance report summary is equivalent to the disbursement on the bank statement
...
 Provides a specification over sales for the given period. The sales total is
-specified per payment area (`CreditCard, Direct Debit, Factoring`) and
+specified per payment area (`CreditCard, Factoring`) and
 underlying payment instruments. Each sales row specify Quantity, Sum sales and
 Amount to pay out, the last one is only eligble **if Swedbank Pay handles the
 Settlement process**.
 
 ##### Fees specification
 
 Provides a specification over fees for the given period. The fees total is
```
Comment (asbjornu) [inline _includes/settlement-reconciliation.md]: This paragraph seems like a duplication of the above "Sales specification" section. Not something that needs to be tackled in this PR, but could we do something about that?

File: payments/index.md [hunk 6]
```diff
@@ -107,7 +105,7 @@ distribution and send it to the consumer.
 
 There are two types of one-phase payments – `sale` and `autoCapture`.
 
-`sale` is used by payment instruments such as Direct Debit and [Swish][swish].
+`sale` is used by payment instruments such as Trustly and [Swish][swish].
```
Comment (asbjornu) [inline payments/index.md]: Trustly is still a work in progress, no?

```suggestion
`sale` is used by payment instruments such as [Swish][swish].
```

File: _includes/settlement-reconciliation.md (hunk 5 above)
Comment (arebra) [reply _includes/settlement-reconciliation.md]: Noted!

**Pull request merged**